### PR TITLE
Remove upper cap for generating effective_cache_size value in postgre…

### PR DESCRIPTION
**Pull request by maciejmrowiec**
…s config.

Set 70% of memory also for larger installations.

Signed-off-by: Maciej Mrowiec <mrowiec.maciej@gmail.com>

### CODE REVIEW

**Comment by maciejmrowiec** [comment]
@kacf @nickanderson also potencial 3.10 candidate if you allow for it.

File: packaging/common/cfengine-hub/postinstall.sh [hunk 1]
```diff
@@ -301,14 +301,18 @@ if [ ! -d $PREFIX/state/pg/data ]; then
...
         shared="16GB"
-        effect="11GB"        #70% of 16G
       else
         shared=$(( $total * 25 / 100 / 1024 ))   #in MB
         shared="$shared""MB"
-        effect=$(( $total * 70 / 100 / 1024 ))   #in MB
-        effect="$effect""MB"
       fi
+
+      # effective_cache_size: 50% of totoal memory is conservicative value
```
Comment (kacf) [inline packaging/common/cfengine-hub/postinstall.sh]: * `s/totoal/total/`
* `s/conservicative/conservative/`

File: packaging/common/cfengine-hub/postinstall.sh [hunk 2]
```diff
@@ -301,14 +301,18 @@ if [ ! -d $PREFIX/state/pg/data ]; then
     if [ "$total" -gt "$lower" ]; then
       maint="2GB"
       if [ "$total" -ge "$upper" ]; then
+        # larger shared buffers provice minior performance improvement
```
Comment (kacf) [inline packaging/common/cfengine-hub/postinstall.sh]: * `s/provice/provide/`
* `s/minior/minor/`

**Comment by maciejmrowiec** [comment]
done

**Comment by kacf** [comment]
Cherry-picked to 3.10.x as well.

**Comment by maciejmrowiec** [comment]
sweet, thanks!

**Pull request merged**